<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_AddFile</class>
 <widget class="QDialog" name="dialog_AddFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add File</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>File path:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_FilePath">
      <property name="placeholderText">
       <string>D:\Documents\MyRequirements.doc</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="toolButton_FilePath">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qt_resource_file.qrc">
        <normaloff>:/actions/ico/16x16/actions/filefind.png</normaloff>:/actions/ico/16x16/actions/filefind.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Regular expression for requirements ID:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_Regex">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>940</y>
     <width>156</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_FilePath</tabstop>
  <tabstop>toolButton_FilePath</tabstop>
  <tabstop>comboBox_Regex</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>fontComboBox</tabstop>
 </tabstops>
 <resources>
  <include location="qt_resource_file.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_AddFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_AddFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
